anti.md

```diff
@@ -58,12 +98,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+You'd have to spend the whole time on the language part because there is no
+systems aspect except following the instructinos on your list.
```

Comment:
To me it seems hard to gauge what the time investment in the systems aspect would look like unless you know what your deliverable will look like (my coffee drink specifying language suffers from the same problem). Since there aren't automated laundry bots that you could program using this language, you'd have to work out some other deliverable, and the nature of the deliverable would affect the amount of time spent on the systems aspect.